queries.sql

```diff
@@ -181,6 +181,10 @@ WHERE vt.name = 'Maisy Smith'
 GROUP BY s.name
 ORDER BY visit_count DESC
 LIMIT 1;
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] 👍 Great job including the screenshots for the `email ` index, both before and after its creation. However, upon closer examination, it appears that the screenshot after the index was created does not utilize the index. In the output, there is no "Index scan," and you can see a "Seq scan" instead. 

| Your | Expected|
| --- | --- |
|  ![image](https://github.com/Kaiserabbas/vet_clinic/assets/55840999/6fa3119e-852f-4da2-94ee-75248b2724ac) | ![image](https://github.com/Kaiserabbas/vet_clinic/assets/55840999/a3fbd585-ba99-429f-8faf-aa4fedfbc60e) |

- [ ] To ensure that the index is functioning as expected, kindly consider following the instructions by removing the indexes and adding them once again. Afterward, take screenshots of the `explain analyze` output one more time to validate the proper utilization of the index. This will help ensure that the index is working as intended. Keep up the good work! 👍🔍📸

Reply:
**Created Index Lits**
![40- Index](https://github.com/Kaiserabbas/vet_clinic/assets/123939543/79e93584-23fb-4387-b3d3-45da2b14f820)

**Re-Run the Queries using Indexes.**
![39- Execution Time After](https://github.com/Kaiserabbas/vet_clinic/assets/123939543/10b3ae8c-8793-4d16-9df8-4de17373a165)